tests/jacobi/include/helpers-jacobi.hpp

```diff
@@ -218,19 +215,20 @@ inline void end_measure(struct counters& timer) {
 }
 
 
-struct Vector_f {
-public:
+struct Vector_f
```

Comment:
What's the use of public: in a struct? (I don't know of any, other than extra LoC. Excuse me, it really bugged me.)